src/main/java/com/gamingmesh/jobs/CMILib/CMIMaterial.java

```diff
@@ -1038,7 +1038,27 @@
 //	LEGACY_PURPUR_DOUBLE_SLAB(204, null, null, "LEGACY_PURPUR_DOUBLE_SLAB", ""),
 //	LEGACY_COMMAND_REPEATING(210, null, null, "LEGACY_COMMAND_REPEATING", ""),
 //	LEGACY_COMMAND_CHAIN(211, null, null, "LEGACY_COMMAND_CHAIN", ""),
-    LEGACY_WHEAT(59, 0, null, "Wheat Block");
+    LEGACY_WHEAT(59, 0, null, "Wheat Block"),
+// Potions
+    AWKWARD("AWKWARD"),
+    FIRE_RESISTANCE("FIRE_RESISTANCE"),
+    INSTANT_DAMAGE("INSTANT_DAMAGE"),
+    INSTANT_HEAL("INSTANT_HEAL"),
+    INVISIBILITY("INVISIBILITY"),
+    JUMP("JUMP"),
+    LUCK("LUCK"),
+    MUNDANE("MUNDANE"),
+    NIGHT_VISION("NIGHT_VISION"),
+    POISON("POISON"),
+    REGEN("REGEN"),
+    SLOW_FALLING("SLOW_FALLING"),
+    SLOWNESS("SLOWNESS"),
+    SPEED("SPEED"),
+    STRENGTH("STRENGTH"),
+    THICK("THICK"),
+    TURTLE_MASTER("TURTLE_MASTER"),
+    WATER_BREATHING("WATER_BREATHING"),
+    WEAKNESS("WEAKNESS");
```

Comment:
Remove these lines from this enum. It is specified in Potion enum class.

Reply:
If I remove this from CMIMaterial enum, tipped_arrow:slowness will only be displayed as "Tipped Arrow" because [this](https://github.com/Zrips/Jobs/pull/802/commits/78cd616f844c5ab0a2fdc08626e182c79ae6cbf7#diff-653e2e5a015b53ecc101fc4e916fe02dR60) will not find a matching name for the meta.

Reply:
Not sure we need to display the tipped arrow effect type. We should put the Potion effect names to the /jobs info when listing out the materials. We should check that if the current material is equal to tipped arrow then it lists out more detail about that.

Reply:
Alright, I changed it with the new commit. 
It should be better to check with isCanHavePotion instead of tipped arrow, so it also works for all other materials defined there